sdks/python/apache_beam/internal/apiclient_test.py

```diff
@@ -35,8 +34,8 @@ def test_create_application_client(self):
 
   def test_default_job_name(self):
     job_name = apiclient.Job.default_job_name(None)
-    regexp = 'beamapp-[a-z]*-[0-9]{10}-[0-9]{6}'
-    self.assertTrue(re.match(regexp, job_name))
+    regexp = 'beamapp-.*-[0-9]{10}-[0-9]{6}'
```

Comment:
As a note, this is also the default behavior in Java, with the only difference that in Python we also lowercase the whole job name.